it/uk/gov/hmrc/fhregistrationfrontend/controllers/BusinessPartnersCorporateBodyUtrControllerISpec.scala

```diff
@@ -0,0 +1,98 @@
+package uk.gov.hmrc.fhregistrationfrontend.controllers
+
+import org.jsoup.Jsoup
+import play.api.libs.ws.DefaultWSCookie
+import play.api.test.WsTestClient
+import uk.gov.hmrc.fhregistrationfrontend.testsupport.{Specifications, TestConfiguration}
+
+class BusinessPartnersCorporateBodyUtrControllerISpec
+  extends Specifications with TestConfiguration {
+
+  val route = "/form/business-partners/corporate-body-corporation-tax-unique-taxpayer-reference"
+
+  s"GET $route" should {
+
+    "render the corporate-body-corporation-tax-unique-taxpayer-reference page" in {
+      given
+        .commonPrecondition
+
+      WsTestClient.withClient { client =>
+        val result = client.url(s"$baseUrl" + s"$route")
+          .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+          .get()
+
+        whenReady(result) { res =>
+          res.status mustBe 200
+          val page = Jsoup.parse(res.body)
+          page.title() must include("What is the company’s Corporation Tax Unique Taxpayer Reference (UTR)?")
+          page.getElementsByTag("h1").text() must include("What is test partner’s Corporation Tax Unique Taxpayer Reference (UTR)?")
+        }
+      }
+    }
+  }
+
+  s"POST $route" when {
+    "The UTR is entered" should {
+      "return 200 with UTR" in {
+        given
+          .commonPrecondition
+
+        WsTestClient.withClient { client =>
+          val result = client.url(s"$baseUrl" + s"$route")
+            .addCookies(DefaultWSCookie("mdtp", authAndSessionCookie))
+            .withHttpHeaders(xSessionId,
+              "Csrf-Token" -> "nocheck")
+            .post(Map("uniqueTaxpayerReference_value" -> Seq("1234567890")))
+
+          whenReady(result) { res =>
+            res.status mustBe 200
+            res.body mustBe "Next page! with UTR: BusinessPartnersEnterUniqueTaxpayerReference(1234567890)"
+          }
+        }
+      }
+    }
+
+
+    "option selected but UTR not supplied" should {
```

Comment:
No option being selected. Please change to remove reference to selecting an option.